.storybook/main.js

```diff
@@ -57,6 +58,11 @@ module.exports = {
         }
       },
       config);
+    // Exclude our build folder from compilation.
+    // This is because shared/site-base build to ES2017
+    // which somehow breaks the FormatJS babel plugin.
+    const babelRule = r.module.rules.find(rule => rule.include?.includes?.(rootFolder));
+    babelRule.exclude = /(node_modules)|(build)/
```

Comment:
This is the build fix.  In our mono-repo, `@thecointech/shared/file` resolves to `/libs/shared/build/file`, and so would end up being re-processed by babel.

This isn't a big deal, but when we shifted `shared/site-base` to ESNext modules (to support root-level await), something broke in the FormatJS parser.  We don't -need- to parse these files, so easiest fix is to prevent Babel from doing that.